client/state/posts/test/selectors.js

```diff
@@ -33,6 +34,25 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getSitePosts()', () => {
```

Comment:
We don't need this for a single test, but I wonder if we should add a util method in createSelector to reset the memo cache.
```javascript
resetCache( getSitePosts );
```